rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+format_code_in_doc_comments = true
+format_macro_bodies = true
+format_macro_matchers = true
```

Comment:
I don't really remember what these two do, but I turned the on on another project.